.eslintrc

```diff
@@ -4,7 +4,7 @@
   "parserOptions": {
     "ecmaVersion": 2018,
     "sourceType": "module",
-    "project": "./tsconfig.json"
+    "project": "./tsconfig.*.json"
```

Comment:
two things:

a) can it lead to perf issues if we scan all of a workspace's tsconfig files? We can potentially have a lot of them 😅 
b) it wouldn't work with `tsconfig.json` files, right? Like in the case of Storybook where we have a `.storybook/tsconfig.json` file

![image](https://user-images.githubusercontent.com/542458/89103007-d700b580-d40e-11ea-9831-4a1d5da3195a.png)

Reply:
1) Eslint scans all directories to find config files. Then each config file will do a glob to find the relative tsconfig near itself. I haven't tried it in a repo with a lot of libraries though. 

There might be some performance hits because originally we only had `tsconfig.json` and eslint would only create one typescript program. With this, it could potentially find 1+ and will create a program for each tsconfig it finds in a directory.

2) That's a good point, we could probably change the glob to `tsconfig.*?.json`

Reply:
> There might be some performance hits because originally we only had tsconfig.json and eslint would only create one typescript program. With this, it could potentially find 1+ and will create a program for each tsconfig it finds in a directory. 

Yeah exactly. that's why I asked. Maybe we can try that on a large repo..maybe the one Victor uses for building the incremental building, just to see if it has a noticeable impact.

> That's a good point, we could probably change the glob to tsconfig.*?.json

yeah, was my thought as well

Reply:
> There might be some performance hits because originally we only had tsconfig.json and eslint would only create one typescript program. With this, it could potentially find 1+ and will create a program for each tsconfig it finds in a directory.

AFAIK there are already performance hits in current state, as current ts root config has no `include`, which means every file is included in compilation. 

But in general more tsconfigs included in eslint project === bigger perf hit. 

NOTE:
We have project that has 4k+ js modules and introducing multiple tsconfigs was a big no no